src/fides/api/service/connectors/sql_connector.py

```diff
@@ -384,8 +386,6 @@ class RedshiftConnector(SQLConnector):
 
     def build_ssh_uri(self, local_address: tuple) -> str:
         """Build SSH URI of format redshift+psycopg2://[user[:password]@][ssh_host][:ssh_port][/dbname]"""
-        config = self.secrets_schema(**self.configuration.secrets or {})
-
```

Comment:
Ah I see, it makes sense for lines 387-388 but removing the other lines is causing some errors

https://github.com/ethyca/fides/actions/runs/9490017910/job/26152564660?pr=4949
https://github.com/ethyca/fides/actions/runs/9490017910/job/26152564893?pr=4949

I think SonarLint is reporting this incorrectly because even though we're not using those parameters for the `SQLConnector` the functions still need to match the base interface.